databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/databricks_run.py

```diff
@@ -10,19 +10,24 @@ def main():
     parser.add_argument("--env", dest="env", type=str)
     parser.add_argument("--conf-source", dest="conf_source", type=str)
     parser.add_argument("--package-name", dest="package_name", type=str)
+    parser.add_argument("--nodes", dest="nodes", type=str)
 
     args = parser.parse_args()
     env = args.env
     conf_source = args.conf_source
     package_name = args.package_name
+    nodes = [node.strip() for node in args.nodes.split(",")]
 
     # https://kb.databricks.com/notebooks/cmd-c-on-object-id-p0.html
     logging.getLogger("py4j.java_gateway").setLevel(logging.ERROR)
     logging.getLogger("py4j.py4j.clientserver").setLevel(logging.ERROR)
 
     configure_project(package_name)
     with KedroSession.create(env=env, conf_source=conf_source) as session:
-        session.run()
+        if len(nodes) == 0:
+            session.run()
+        else:
+            session.run(node_names=nodes)
```

Comment:
A couple of questions:
- Do we not need to support the full CLI? `--params` feels like something at the very least we need to cover in this.
- Is there a way we can reuse any of our click stuff? In an ideal world we'd use only `argparse` but Kedro predates it being added to the `stdlib`.

Reply:
The only reason we have a databricks specific run file is click, and I think we have a solution now to solve this consistently so there will be one entrypoint for all starters. It supports the full kedro run API so I think we can leave the CLI argument discussion

Reply:
If i understand you correctly, it means that we could use the built-in entry point for run instead of having the run script at all?

Reply:
@JenspederM Yes, but this is out of scope for this PR. The PR is still in progress but I think we are getting close.